operate/client/src/modules/stores/processInstanceMigrationMapping.test.ts

```diff
@@ -20,62 +20,6 @@ jest.mock('modules/stores/processes/processes.migration', () => ({
   },
 }));
 
-/**
```

Comment:
I removed this, because it does not add much value in my opinion

Reply:
I agree.